packages/requisitions/src/RequestRequisition/api/hooks.ts

```diff
@@ -129,6 +130,19 @@ export const useRequestFields = <
   );
 };
 
+export const useRequestLineChartData = (requisitionLineId: string) => {
+  const api = useRequestApi();
+  return useQuery(
+    api.keys.chartData(requisitionLineId),
+    () => api.get.lineChartData(requisitionLineId),
+    {
+      refetchOnMount: false,
+      cacheTime: 0,
+      onError: () => {},
```

Comment:
I've added a generic error handler too.. and if that goes in, we don't want to be notified for the chart data, just silently not show the charts 😁